src/client/app/components/about.ts

```diff
@@ -50,28 +63,75 @@ export class About implements AfterViewInit {
+    ) {}
+
     ngAfterViewInit() {
         this.availableLanguages = getAvailableLanguages();
         this.currentChosenLanguage = getDisplayLanguage();
         this.originalLanguage = this.currentChosenLanguage;
+        this.configs.push(
+            { name: 'Production', value: 'production' },
+            { name: 'Beta', value: 'insiders' },
+            { name: 'Alpha', value: 'edge' },
+        );
+
+        // User can only navigate to localhost if they've sideloaded local manifest
+        let showLocalConfig = environment.current.config.name === config['local'].name || (window.localStorage.getItem(PLAYGROUND_ORIGIN) && window.localStorage.getItem(PLAYGROUND_ORIGIN).indexOf('localhost') !== -1);
+        if (showLocalConfig) {
+            this.configs.push({ name: config['local'].editorUrl, value: 'local' });
+        }
+
+        this.selectedConfig = this.configs.find(c => c.value.toUpperCase() === environment.current.config.name).value;
     }
 
-    okClicked() {
+    async okClicked() {
         if (this.currentChosenLanguage !== this.originalLanguage) {
             setDisplayLanguage(this.currentChosenLanguage);
             window.location.reload();
         }
 
         this.showChange.emit(false);
-    }
 
-    cache = `
-    ${Strings().aboutStorage}
-    ${storageSize(localStorage, `playground_${environment.current.host}_snippets`, Strings().aboutSnippets)}
-    ${storageSize(sessionStorage, 'playground_intellisense', Strings().aboutIntellisense)}
-    `;
+        let currentConfigName = environment.current.config.name.toLowerCase();
+        if (this.selectedConfig === currentConfigName) {
+            return;
+        }
+
+        let result = await this._effects.alert(
+            this.strings.changeEnvironmentConfirm,
+            `${this.strings.aboutSwitchEnvironment} ${this.configs.find(c => c.value === currentConfigName).name } to ${this.configs.find(c => c.value === this.selectedConfig).name }`,
+            this.strings.okButtonLabel,
+            this.strings.cancelButtonLabel
+        );
+        if (result === this.strings.cancelButtonLabel) {
+            this.selectedConfig = this.configs.find(c => c.value === currentConfigName).value;
+            this._changeDetector.detectChanges();
+            return;
+        }
+
+        let originEnvironment = window.localStorage.getItem(PLAYGROUND_ORIGIN);
+        let targetEnvironment = config[this.selectedConfig].editorUrl;
+
+        // Add query string parameters to default editor URL
+        if (originEnvironment) {
+            window.location.href = originEnvironment + '?targetEnvironment=' + encodeURIComponent(targetEnvironment);
+        } else {
+            window.localStorage.setItem(PLAYGROUND_REDIRECT, targetEnvironment);
+            window.location.href = targetEnvironment + '?originEnvironment=' + encodeURIComponent(environment.current.config.editorUrl);
+        }
+    }
 
     setLanguage(languageCode: string) {
```

Comment:
These functions are actually not needed. The ngModel binding in the `select` tag automatically updates `selectedConfig` (and `currentChosenLanguage` in the above `select`). Since the 'okClicked()' function is actually handling any events that we need, both functions are superfluous.